seatunnel-dist/release-docs/LICENSE

```diff
@@ -390,15 +390,21 @@ The text of each license is the standard Apache 2.0 license.
      (Apache License, Version 2.0) Apache HBase - Spark Connector (org.apache.hbase.connectors.spark:hbase-spark:1.0.0 - http://hbase.apache.org/spark/hbase-spark)
      (Apache License, Version 2.0) Apache HBase - Zookeeper (org.apache.hbase:hbase-zookeeper:2.0.0 - http://hbase.apache.org/hbase-build-configuration/hbase-zookeeper)
      (Apache License, Version 2.0) Apache HBase - Zookeeper (org.apache.hbase:hbase-zookeeper:2.1.0 - http://hbase.apache.org/hbase-build-configuration/hbase-zookeeper)
+     (Apache License, Version 2.0) Apache Hadoop Annotations (org.apache.hadoop:hadoop-annotations:2.7.2 - no url defined)
```

Comment:
It seems we don't need to add `hadoop-annotations:2.7.2`, `hadoop-annotations:2.7.7` to the LICENSE file. Please check this file, some of the dependencies are not in our distribution.

Reply:
@ruanwenjun  Yes, I remove `hadoop-annotations:2.7.2` and `hadoop-annotations:2.7.7` from LICENSE file.

Reply:
And some other Hadoop-xx also need to delete.

Reply:
@ruanwenjun  done.